class/abstract-hd-segwit-p2sh-wallet.js

```diff
@@ -90,7 +135,7 @@ export class AbstractHDSegwitP2SHWallet extends AbstractHDWallet {
    */
   async _getWIFByIndex(index) {
     if (!this.seed) {
-      this.seed = await bip39.mnemonicToSeed(this.secret);
+      this.seed = await this.getSeed();
```

Comment:
`this.seed()` doesn't return Promise so `await` isn't required here

Reply:
it does return promise